dart/lib/src/attachment_extensions/attachment_extensions.dart

```diff
@@ -0,0 +1,2 @@
+export '_io_attachment_extensions.dart'
+    if (dart.library.html) '_web_scope_extensions.dart';
```

Comment:
sometimes we check for `html` and sometimes for `if (dart.library.io)`, lets standardized and use the same check aways, just a nit though